Automattic/wp-calypso
# Add Tracks events to searches in the block picker

This sends events to Tracks when users perform searches in the block selector in Gutenberg.

#### Changes proposed in this Pull Request

* Sends two new events to Tracks. One for every search that is performed in the block picker and another for when the search doesn't return any results. I thought it was best to create two events as its hard to filter an event by two properties in Tracks.

This is part of paAmJe-lV-p2

#### Testing instructions
- Get this code running on your sandbox (see field guide)
- Go to the block editor
- Turn on debugging by running this in your browser console: `localStorage.setItem('wpcom-block-editor:analytics:tracks');`
-Do some searches and check that they are logged in your console. You should see two events when there are no results.
